inst/sass-color/app.R

```diff
@@ -0,0 +1,65 @@
+library(shiny)
+library(sass)
+library(colourpicker)
+
+input_scss <- "new-style.scss"
+output_css <- "new-style.css"
```

Comment:
Let's not write the output to a file on disk--it doesn't work well with multiple sessions connected to the same process, and doesn't work at all on Connect (app dirs are read-only there).